genie-web/src/main/java/com/netflix/genie/web/tasks/leader/ClusterCheckerTask.java

```diff
@@ -121,42 +121,44 @@ public void run() {
             .filter(host -> !this.hostName.equals(host))
             .forEach(this::validateHostAndUpdateErrorCount);
 
-        this.errorCounts.entrySet().removeIf(entry -> {
-            final String host = entry.getKey();
-            boolean result = true;
-            if (entry.getValue() >= properties.getLostThreshold()) {
-                try {
-                    updateJobsToFailedOnHost(host);
-                } catch (Exception e) {
-                    log.error("Unable to update jobs on host {} due to exception", host, e);
-                    unableToUpdateJobCounter.increment();
+        this.errorCounts.entrySet().removeIf(
+            entry -> {
+                final String host = entry.getKey();
+                boolean result = true;
+                if (entry.getValue() >= this.properties.getLostThreshold()) {
+                    try {
+                        this.updateJobsToFailedOnHost(host);
+                    } catch (Exception e) {
+                        log.error("Unable to update jobs on host {} due to exception", host, e);
+                        this.unableToUpdateJobCounter.increment();
+                        result = false;
+                    }
+                } else {
                     result = false;
                 }
-            } else {
-                result = false;
+                return result;
             }
-            return result;
-        });
+        );
         log.info("Finished checking for cluster node health.");
     }
 
     private void updateJobsToFailedOnHost(final String host) {
-        final Set<Job> jobs = jobSearchService.getAllActiveJobsOnHost(host);
+        final Set<Job> jobs = this.jobSearchService.getAllActiveJobsOnHost(host);
```

Comment:
this.jobSearchService replaces jobSearchService in a lot of places. Is this needed/intentional?

Reply:
Needed? No.

Intentional? Yes. For consistency across the codebase and to make it clear what is a local variable vs. a member variable reference member variables/functions are referenced using `this.`